sources/packages/backend/apps/api/src/services/application-exception/application-exception.service.ts

```diff
@@ -214,7 +214,7 @@ export class ApplicationExceptionService extends RecordDataModelService<Applicat
         .andWhere(
           new Brackets((qb) => {
             qb.where(
-              "CONCAT(user.firstName,' ', user.lastName) Ilike :searchCriteria",
+              "(user.firstName || ' ' || user.lastName) Ilike :searchCriteria",
```

Comment:
We can have a simple util method to create this string in a centralized way
```ts
export function getUserFulNameLikeSearch(userTableAlies = "user", searchCriteriaParameteName = "searchCriteria"){
    return `(${userTableAlies }.firstName || ' ' || ${userTableAlies }.lastName) ILIKE :${searchCriteriaParameteName }`;
}
```

Reply:
Thanks for suggesting this change.